src/Parts/Emoji.php

```diff
@@ -9,7 +9,7 @@ class Emoji
     public ?string $id;
     public ?string $name;
     /**
-     * @var \Ragnarok\Fenrir\Parts\Role[]
+     * @var Role[]
```

Comment:
3rd party json mapper requires complete paths in doc blocks (not blaming you for not knowing about this limitation)